aws-crt-kotlin/common/src/aws/sdk/kotlin/crt/http/HttpClientConnectionManager.kt

```diff
@@ -14,6 +14,11 @@ import aws.sdk.kotlin.crt.Closeable
 public expect class HttpClientConnectionManager(options: HttpClientConnectionManagerOptions) :
     Closeable,
     AsyncShutdown {
+    /**
+     * The active metrics for this connection manager
+     */
+    public val managerMetrics: HttpManagerMetrics
```

Comment:
**nit**: Wouldn't `metrics` suffice here given it's namespaced under `HttpClientConnectionManager`

Reply:
Similar to other Java→Kotlin mapping classes, I chose to keep this [the same as the underlying property](https://github.com/awslabs/aws-crt-java/blob/3eea28d5f617f408655054efd33278ce84d0563b/src/main/java/software/amazon/awssdk/crt/http/HttpClientConnectionManager.java#L219-L227). If you feel strongly I can change it to be simpler (which I would generally prefer when not mapping existing classes).